standalone/runtime/src/lib.rs

```diff
@@ -21,59 +21,60 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 // `construct_runtime!` does a lot of recursion and requires us to increase the limit to 256.
 #![recursion_limit = "256"]
-
 #![allow(clippy::identity_op)]
 
 mod msg_routing;
 
 use codec::{Decode, Encode, MaxEncodedLen};
 use frame_election_provider_support::{
-	onchain, BalancingConfig, ElectionDataProvider, SequentialPhragmen, VoteWeight,
+    onchain, BalancingConfig, ElectionDataProvider, SequentialPhragmen, VoteWeight,
```

Comment:
This file shouldn't be formatted with soft tab. It's modified from substrate original code, which uses the hard tab.